content/docs/command-reference/import.md

```diff
@@ -276,3 +278,133 @@ outs:
 
 The `url` and `rev_lock` subfields under `repo` are used to save the origin and
 [version](https://git-scm.com/docs/revisions) of the dependency, respectively.
+
+## Example: Chained imports
+
+DVC supports importing data importing data that was imported into the source
```

Comment:
"importing data importing data that was imported" looks fun 😄

Reply:
Probably just a typo. Fixing...